core/app/views/spree/order_mailer/inventory_cancellation_email.html.erb

```diff
@@ -0,0 +1,26 @@
+<table>
+  <tr>
+    <td>
+      <p class="lede">
+        <%= Spree.t('order_mailer.inventory_cancellation.dear_customer') %>
+      </p>
+      <p>
+        <%= Spree.t('order_mailer.inventory_cancellation.instructions') %>
+      </p>
+      <p>
+        <%= Spree.t('order_mailer.inventory_cancellation.order_summary_canceled') %>
+      </p>
+      <table>
+        <% @inventory_units.each do |item| %>
+          <tr>
+            <td><%= item.variant.sku %></td>
+            <td><%= raw(item.variant.product.name) %></td>
+            <td><%= raw(item.variant.options_text) -%></td>
```

Comment:
I used `raw`  because i copied from https://github.com/solidusio/solidus/blob/c5b9e0242291a2249ea62b046cc973b530957d51/core/app/views/spree/order_mailer/confirm_email.html.erb#L18

Probably `sanitize` is better, i change also in other templates.